docstring/src/test/java/io/cucumber/docstring/DocStringTypeRegistryDocStringConverterTest.java

```diff
@@ -16,49 +19,47 @@
         registry);
 
     @Test
-    void uses_doc_string_type_when_available() {
-        registry.defineDocStringType(new DocStringType(
-            JsonNode.class,
-            "json",
-            (String s) -> new ObjectMapper().readTree(s)));
-
+    void target_type_to_string_is_predefined() {
         DocString docString = DocString.create(
-            "{\"hello\":\"world\"}",
-            "json");
+            "hello world");
 
-        JsonNode converted = converter.convert(docString, Object.class);
-        assertThat(converted.get("hello").textValue(), is("world"));
+        String converted = converter.convert(docString, String.class);
+        assertThat(converted, is("hello world"));
     }
 
     @Test
-    void uses_target_type_when_available() {
-        registry.defineDocStringType(new DocStringType(
-            JsonNode.class,
-            "json",
-            (String s) -> new ObjectMapper().readTree(s)));
-
+    void converts_doc_string_to_doc_string() {
         DocString docString = DocString.create(
             "{\"hello\":\"world\"}");
 
-        JsonNode converted = converter.convert(docString, JsonNode.class);
-        assertThat(converted.get("hello").textValue(), is("world"));
+        DocString converted = converter.convert(docString, DocString.class);
+        assertThat(converted, is(docString));
     }
 
     @Test
-    void target_type_to_string_is_predefined() {
+    void converts_no_content_type_doc_string_to_registered_matching_convertor() {
         DocString docString = DocString.create(
-            "hello world");
-        String converted = converter.convert(docString, String.class);
-        assertThat(converted, is("hello world"));
+            "{\"hello\":\"world\"}");
+
+        registry.defineDocStringType(new DocStringType(
+            JsonNode.class,
+            "json",
+            (String s) -> new ObjectMapper().convertValue(s, JsonNode.class)));
+
+        JsonNode converted = converter.convert(docString, JsonNode.class);
+        assertThat(converted.asText(), equalTo(docString.getContent()));
     }
 
     @Test
-    void converts_doc_string_to_doc_string() {
-        DocString docString = DocString.create(
-            "{\"hello\":\"world\"}");
+    void throws_when_no_converter_defined() {
```

Comment:
This test does not involve the converter. I think it should move back to `DocStringTest`.

Reply:
You are right, moving it back to it's original place.